client/settings/deposits/index.js

```diff
@@ -0,0 +1,217 @@
+/**
+ * External dependencies
+ */
+import React, { useContext } from 'react';
+import { __, sprintf } from '@wordpress/i18n';
+import {
+	Card,
+	SelectControl,
+	ExternalLink,
+	Notice,
+} from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import WCPaySettingsContext from '../wcpay-settings-context';
+import CardBody from '../card-body';
+import {
+	useDepositScheduleInterval,
+	useDepositScheduleWeeklyAnchor,
+	useDepositScheduleMonthlyAnchor,
+	useDepositDelayDays,
+	useDepositStatus,
+	useCompletedWaitingPeriod,
+} from '../../data';
+import './style.scss';
+
+const CustomizeDepositSchedule = () => {
+	const [
+		getDepositScheduleInterval,
+		setDepositScheduleInterval,
+	] = useDepositScheduleInterval();
+	const [
+		getDepositScheduleWeeklyAnchor,
+		setDepositScheduleWeeklyAnchor,
+	] = useDepositScheduleWeeklyAnchor();
+	const [
+		getDepositScheduleMonthlyAnchor,
+		setDepositScheduleMonthlyAnchor,
+	] = useDepositScheduleMonthlyAnchor();
+
+	const daysOfWeek = [
```

Comment:
**Nitpick:** `daysOfWeek` and `monthlyAnchors` can be defined outside of the component as they do not use any props or state.